packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form-side-note.tsx

```diff
@@ -25,119 +25,77 @@ const AccountTransferBullet = ({ children }: React.PropsWithChildren) => (
+        if (transfer_fee === 2) {
             return (
                 <Localize
-                    i18n_default_text='We’ll charge a 2% transfer fee or {{minimum_fee}} {{currency}}, whichever is higher, for transfers between your Deriv cryptocurrency accounts.'
-                    values={{
-                        minimum_fee,
-                        currency: getCurrencyDisplayCode(currency),
-                    }}
-                />
-            );
-        } else if (transfer_fee === 2 && (is_mt_transfer || is_dxtrade_transfer)) {
-            return is_dxtrade_allowed ? (
-                <Localize
-                    i18n_default_text='We’ll charge a 2% transfer fee or {{minimum_fee}} {{currency}}, whichever is higher, for transfers between your Deriv cryptocurrency and Deriv MT5 accounts and between your Deriv cryptocurrency and {{platform_name_dxtrade}} accounts.'
-                    values={{
-                        minimum_fee,
-                        currency: getCurrencyDisplayCode(currency),
-                        platform_name_dxtrade,
-                    }}
-                />
-            ) : (
-                <Localize
-                    i18n_default_text='We’ll charge a 2% transfer fee or {{minimum_fee}} {{currency}}, whichever is higher, for transfers between your Deriv cryptocurrency and Deriv MT5 accounts.'
+                    i18n_default_text='We charge 2% or {{minimum_fee}} {{currency}} (whichever is higher) for all cryptocurrency transfers.'
                     values={{
                         minimum_fee,
                         currency: getCurrencyDisplayCode(currency),
                     }}
                 />
             );
-        } else if (transfer_fee === 2 && !is_mt_transfer && !is_dxtrade_transfer) {
+        } else if (transfer_fee === 1) {
             return (
                 <Localize
-                    i18n_default_text='We’ll charge a 2% transfer fee or {{minimum_fee}} {{currency}}, whichever is higher, for transfers between your Deriv fiat and Deriv cryptocurrency accounts.'
+                    i18n_default_text='We charge 1% or {{minimum_fee}} {{currency}} (whichever is higher) for all cryptocurrency transfers.'
                     values={{
                         minimum_fee,
                         currency: getCurrencyDisplayCode(currency),
                     }}
                 />
             );
         }
-        return null;
-    }, [
-        currency,
-        is_crypto_to_crypto_transfer,
-        is_dxtrade_allowed,
-        is_dxtrade_transfer,
-        is_mt_transfer,
-        minimum_fee,
-        platform_name_dxtrade,
-        platform_name_mt5,
-        platform_name_ctrader,
-        transfer_fee,
-    ]);
+        return (
+            <Localize i18n_default_text='No fees for transfer between fiat account (with the same currency) to Deriv MT5, and Deriv X account(s), vice versa.' />
```

Comment:
cTrader is not available in `daily_cumulative_amount_transfers`, not sure if it should be displayed in the notes. I'll remove it from the `getPlatformsAllowedNotes` check to avoid confusion

<img width="527" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/90be5d9c-ce3b-41d0-ad0f-fce1a583f124">